src/viking.js

```diff
@@ -1,11 +1,98 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+    }
+
+    // Iteration 1: Soldier
+attack() {
+    return this.strength;
+};
+
+receiveDamage(damage) {
+    this.damage = damage;
+    this.health = this.health - damage;
+    }
+};
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength) {
+    super(name, health, strength);
```

Comment:
Here you would just pass what is needed for the parent's constructor 
```suggestion
    super(health, strength);
```